lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/MethodGenerator.java

```diff
@@ -58,24 +58,24 @@ public abstract List<String> generate(
   protected String methodSigDef(String owner, List<MethodParameter> params, boolean isAbstract) {
     int paramsLen = params.size();
     String paramsDef;
+    boolean includeSelfParam = !(isStatic || isConstructor);
```

Comment:
No? This particular change affects generating builtin nodes.
For example (roughly) for
```
@Builtin.Method
public static void foo(Object x) {
  // ...
}
```
it will generate Builtin node with `execute` without `Object self` i.e.
```

@BuiltinMethod(...)
public class FooNode {
    public void execute(Object x) {
        /// ...
    }
}
```
rather than 
```
@BuiltinMethod(...)
public class FooNode {
    public void execute(Object self, Object x) {
        /// ...
    }
}
```
This is then accepted by the next processor which knows that it needs to add `self` as ArgumentDefinition at 0.
While strictly speaking not necessary, because in this version everything still has `self` ( <insert sad face> ) but at least most of it is now hidden.